src/common/naming/common_naming/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+
+LETTERS = re.compile('[^a-zA-Z]')
```

Comment:
Maybe use [string.acii_letters](https://docs.python.org/fr/3/library/string.html#string.ascii_letters) ?

Reply:
How would you use `ascii_letts`?

Reply:
Nevermind, you use it to remove all non-letters occurences. The variable name confused me